_opensearch/rest-api/create-alias.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Create alias
```

Comment:
Hey @keithhc2, I believe this method + path combination (POST _aliases) is actually the "add or remove aliases" operation, so calling the page "Create alias" isn't quite right.